archive-product.php

```diff
@@ -11,33 +11,31 @@
     <div class="row margin-bottom-small">
 <?php
 if( have_posts() ) {
-  $i = 0;
   while( have_posts() ) {
     the_post();
 
     $product = new WC_Product($post->ID);
     $in_stock = $product->is_in_stock();
     $availability = $product->get_availability();
 
-    if ($i % 4 === 0 && $i !== 0) {
-      echo '</div>\n<div class="row margin-bottom-small">';
```

Comment:
because on tablet its 3 to a row and on mobile its 1. we can just let flex wrap the items and use the margin-bottom on the items instead of the row